optuna/pruners/_hyperband.py

```diff
@@ -1,9 +1,9 @@
+from __future__ import annotations
+
 import binascii
 import math
 from typing import Container
```

Comment:
I think this should be `collections.abc.Container` rather than `typing.Container` as explained by the original issue.